packages/test/local-server-tests/src/test/containerHydration.spec.ts

```diff
@@ -0,0 +1,175 @@
+/*!
```

Comment:
I do not think adding this test to local-server-tests folder is the right way to go.
I think test/test-end-to-end-tests is better place for it.